CODEOWNERS

```diff
@@ -5,9 +5,6 @@
 
 /pkg/types/		@bobcallaway
 
-/pkg/api/timestamp.go	@asraa @loosebazooka
-/pkg/types/rfc3161/	@asraa @loosebazooka
```

Comment:
keep the rfc type?

Reply:
Whoops, I'll create a PR to fix this, sorry!